docs/recipes/IsolatingSubapps.md

```diff
@@ -0,0 +1,81 @@
+# Isolating Redux Subapps
+
+Consider the case of a “big” app (contained in a `<BigApp>` component)
+that embeds smaller “subapps” (contained in `<SubApp>` components):
+
+```js
+import React, { Component } from 'react'
+import SubApp from './subapp'
+
+class BigApp extends Component {
+  render() {
+    return (
+      <div>
+        <SubApp />
+        <SubApp />
+        <SubApp />
+      </div>
+    )
+  }
+}
+```
+
+These `<SubApp>`s will be completely independent. They won’t share data or
+actions, and won’t see or communicate with each other.
+
+It’s best not to mix this approach with standard Redux reducer composition.
+For typical unified web products, stick with reducer composition. For
```

Comment:
"unified web products" sounds a little close to "universal web apps", so that might be confusing to newbies. I'd just say "For typical web apps". Thoughts?